lib/chalk-log.rb

```diff
@@ -36,6 +36,13 @@ module Chalk::Log
   # add.)
   LEVELS = [:debug, :info, :warn, :error, :fatal].freeze
 
+  module CLevels
```

Comment:
In this instance, the values should be symbols instead of strings. We try to use symbols every for internal identifiers.